packages/core/src/components/forms/fileInput.tsx

```diff
@@ -11,28 +11,28 @@ import { Utils } from "../../common";
 import * as Classes from "../../common/classes";
 import { IProps } from "../../common/props";
 
-export interface IFileUploadProps extends React.AllHTMLAttributes<HTMLLabelElement>, IProps {
+export interface IFileInputProps extends React.AllHTMLAttributes<HTMLLabelElement>, IProps {
     /**
-     * Whether the file upload is non-interactive.
+     * Whether the file input is non-interactive.
      * Setting this to `true` will automatically disable the child input too.
      */
     disabled?: boolean;
 
     /**
-     * Whether the file upload should take up the full width of its container.
+     * Whether the file input should take up the full width of its container.
      */
     fill?: boolean;
 
     /**
      * The props to pass to the child input.
      * `disabled` will be ignored in favor of the top-level prop.
      * `type` will be ignored, because the input _must_ be `type="file"`.
-     * Pass `onChange` here to be notified when the user uploads a file.
+     * Pass `onChange` here to be notified when the user inputs a file.
```

Comment:
I think `uploads` still makes sense here?

Reply:
Should it be "selects" instead? Since I don't think the file actually gets uploaded, but inputs is probably wrong.

Reply:
"select" is definitely preferable to completely incorrect "upload".

the problem with the old name `file-upload` is that there's no actual uploading in this component!